### ISSUE TITLE
[20220109] Weekly AI ArXiv 만담 - 35회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#35]
- News
  - 모더레이터 분들의 2021 가장 기억에 남는 논문 혹은 소식 한개씩! (두개 안됨!)
  - ACL 2022 (ARR-11월) 리뷰가 화요일 즈음 나옵니당.
  - IJCAI 2022 데드라인: 1월 14일 / ICML 2022 데드라인: 1월 20일&27일 
  - CES 2022 !!!
    - 카이스트 교수님들의 시각: https://www.youtube.com/watch?v=7cU7jddtfQA
    - 애널리스트들의 시각: https://www.youtube.com/watch?v=yCCNN88rWV0
  - [AIAI2022 Workshop](https://aiworkshop.kr/)
  
- Arxiv
  - [Towards a Unified Foundation Model: Jointly Pre-Training Transformers on Unpaired Images and Text](https://arxiv.org/abs/2112.07074) 
    - 구글리서치에서 나온 image-text foundation model
    - Joint training 을 하는데 shared transformer, 각각의 task로 multitask pretraining 하면 vision only & task only 둘다 성능 향상
    - KD (Pretrained BERT와 Pretrained ViT를 teacher로 활용) + gradient masking
    - 그런데 multimodal task 실험은 없음: VQA, SNLI-VE, I2T 같은 것들.
![image](https://user-images.githubusercontent.com/11782739/148670184-2f561b5b-8732-458b-a6c3-e1b116b0b412.png)
![image](https://user-images.githubusercontent.com/11782739/148670267-d1e19f7e-ccee-4d25-8b50-fc5d0be00ae1.png)

  - [ERNIE 3.0 Titan: Exploring Larger-scale Knowledge Enhanced Pre-training for Language Understanding and Generation](https://arxiv.org/abs/2112.12731)
    - Baidu Ernie 3.0의 260B 버전 - Text + KG 도 함께 학습
    - self-supervised adversarial loss and a controllable language modeling loss + KD인데 teacher가 self-training도 동시에 하는구조
    - NPU, 서빙을 위한 여러 테크닉 설명까지 포함.
![image](https://user-images.githubusercontent.com/11782739/148670334-51ab8b1b-1590-4426-852b-f5dc0930c795.png)
![image](https://user-images.githubusercontent.com/11782739/148670344-40c35403-92d4-431f-bda1-466f59a3f706.png)

  - [GLIDE: Towards Photorealistic Image Generation and Editing with Text-Guided Diffusion Models](https://arxiv.org/abs/2112.10741)
    - Open AI 에서 공개한 Diffusion model 기반의 Text-to-image 생성 + 편집 모델
    - Classifier-free & CLIP 기반 guidance로 이미지 컨트롤 가능, prompt 와 segmentation mask로 부분편집됨.
    - https://github.com/openai/glide-text2im 
    - 역시 이미지 퀄은 diffusion model ! A100 한장으로 인퍼런스 15초 걸리는 문제만 해결하면 참 좋을 텐데.
![image](https://user-images.githubusercontent.com/11782739/148677909-b5b77530-b9f8-4ec7-9742-62048fff2bcb.png)
![image](https://user-images.githubusercontent.com/11782739/148677928-c5b3cf40-0f1a-411d-b4c3-7a3cf1e97c05.png)

  - [A Neural Network Solves and Generates Mathematics Problems by Program Synthesis: Calculus, Differential Equations, Linear Algebra, and More](https://arxiv.org/abs/2112.15594)
    - MIT, Havard U, Columbia U, U of Warterloo 연구자들이 만든 대학수학을 푸는 AI
    - Open AI CODEX를 이용한 수학문제 --> 코드 (신박한 아이디어)
![image](https://user-images.githubusercontent.com/11782739/148679423-d98fb0f6-c5c7-498d-8080-d6fccd766a97.png)
![image](https://user-images.githubusercontent.com/11782739/148679428-e68a1eec-c8a0-437d-ac5c-1c822fb03df4.png)

**Comment by winterzzi** [comment]
한달에 한번은 발표 하는게 목표에요 'ㅅ'...!
제가 잘못알고 있는 부분이나 추가 설명 해주시는 부분은 언제나 환영합니다~!

1. Medical Article
- https://www.thelancet.com/journals/landig/article/PIIS2589-7500(21)00228-4/fulltext
- IF 24.519 인 란셋 디지털 헬스에서 01JAN2022 에 나온 따끈따끈한 논문입니다.
- Summary
![summary_medpr03](https://user-images.githubusercontent.com/56566861/148157612-0e384e82-779a-46e8-9e95-2c0ae5b1904b.JPG)

- 방법적으로는 다소 올드하고 평범. backpropagation algorithm 사용했다고 하고.. 논문에는 hyperparameter tuning 이나 모델에 대한 디테일한 설명도 없음. appendix 에 gradient boosting, random forest, logistic regression 을 사용했다고 언급되어 있지만 자세한 내용은 없음.
- 심지어 STATA 사용
- 데이터가 깡패
- Model : SWEDHEART(The swedish Web system for Enhancement and Development for Evidence-based care in Heart disease Evaluated According to Recommended Therapies) : n=139,288 (https://www.ucr.uu.se/swedeheart/)
- External validation : WDHR(Western Denmark Heart Registry) n = 30,971

역시 데이터가..중요하다...

**Comment by ghlee0304** [comment]
- Arxiv
    - [IQDUBBING: Prosody modeling based on discrete self-supervised speech representation for expressive voice conversion](https://arxiv.org/abs/2201.00269)
    - [Multi-speaker Multi-style Text-to-speech Synthesis With Single-speaker Single-style Training Data Scenarios](https://arxiv.org/abs/2112.12743)
    - [Multi-Singer: Fast Multi-Singer Singing Voice Vocoder With A Large-Scale Corpus](https://arxiv.org/abs/2112.10358)

**Comment by nick-jhlee** [comment]
- COLT 2022 deadline: 02/09

- *새로운* conference/journal venue!: **Transaction on Machine Learning Research (TMLR; https://jmlr.org/tmlr/)**
  - Creator: Hugo Larohelle, Raia Hadsell, Kyunghyun Cho and others
  - **JMLR-style journal**
  - Uses OpenReview
  - Focuses on conference-length publications
  - **Has no submission deadlines**
  - Aims for a fast turnaround
  - **Acceptance based on matched claims and evidence, not potential impact**
  - 더 자세한 launching 내용은 여기로: https://medium.com/@hugo_larochelle_65309/announcing-the-transactions-on-machine-learning-research-3ea6101c936f
한번 보시고, 여기에 제출해보는 것도 하나의 방안일수도 있을 것 같습니당...!
(첫 submission은 대략 2022 초반으로 예상한다고 합니당)

**Comment by nick-jhlee** [comment]
- Very *incomplete* recap (papers):
  - Theory (interpolation/extrapolation, generalization of DNN):
    - [Learning in High Dimension Always Amounts to Extrapolation](https://arxiv.org/pdf/2110.09485.pdf)
    - [A Universal Law of Robustness via Isoperimetry](https://openreview.net/pdf?id=z71OSKqTFh7)
  - Rest of the Theory (inner workings of SGD, GNN, RL, bandits): 굳이 적진 않지만, 아주 많았습니다,,, ㅎㅎ
  - Peer review is broken:
    - [Inconsistency in Conference Peer Review: Revisiting the 2014 NeurIPS Experiment](https://arxiv.org/pdf/2109.09774.pdf)
    - [You Are the Best Reviewer of Your Own Papers: An Owner-Assisted Scoring Mechanism](https://openreview.net/pdf?id=xmx5rE9QP7R)
    - NeurIPS 2021 experiment도 기대...!
    - cf. https://docs.google.com/document/d/1j7Mn2ZkquSzWJ_EzxdXBP3z_JQtrSeUa-CQ0gotAuYw/edit
    - ICML 2021 "...we plan to reduce the number of accepted papers. Please work with your SAC to raise the bar. AC/SAC do not have to accept a paper only because there is nothing wrong in it."
    - AAAI... 할많하않 (IJCAI도 two-phase던데,,)
  - Geometric Deep Learning: [Geometric Deep Learning: Grids, Groups, Graphs, Geodesics, and Gauges](https://arxiv.org/pdf/2104.13478.pdf)
    - https://geometricdeeplearning.com/
    - ICLR 2021 tutorial by Prof. Michael Bronstein: https://www.youtube.com/watch?v=w6Pw4MOzMuo
  - ML/DL in natural sciences
    - biology: [Highly accurate protein structure prediction with AlphaFold](https://www.nature.com/articles/s41586-021-03819-2.pdf)
    - chemistry: [Pushing the frontiers of density functionals by solving the fractional electron problem](https://www.science.org/doi/epdf/10.1126/science.abj6511)
    - (pure) mathematics: [Advancing mathematics by guiding human intuition with AI](https://www.nature.com/articles/s41586-021-04086-x.pdf)
    - teaching: [A Neural Network Solves and Generates Mathematics Problems by Program Synthesis: Calculus, Differential Equations, Linear Algebra, and More](https://arxiv.org/pdf/2112.15594.pdf)
    
- Very *incomplete* recap (news):
  - 1/2 of Nobel Physics to Giorgi Parisi!
  - RIP Naftali Tishby (coauthor of [Opening the black box of Deep Neural Networks via Information](https://arxiv.org/pdf/1703.00810.pdf))

**Comment by nick-jhlee** [comment]
cf. Pedro Domingos의 twitter war
"Half of the female STEM faculty in the US were hired over more qualified men." - Pedro Domingos (01/01/2022)

**Comment by kimyoungdo0122** [comment]
- News
    - 21년 올해의 뉴스
    - [‘초거대 AI’ 기술 전쟁 본격화… 美·中 선두경쟁, 추격하는 韓](https://news.naver.com/main/read.naver?mode=LSD&mid=shm&sid1=105&oid=366&aid=0000780703)
    - [통신3사, AI스피커 2.0 시대 연다...SKT 'AI플랫폼', KT 'B2B', LGU+ '홈IoT'](https://www.techm.kr/news/articleView.html?idxno=92955)
    -

**Comment by hollobit** [comment]
AI 의료기기 성능평가 국제표준화 시동 
- https://search.daum.net/search?w=tot&DA=23A&rtmaxcoll=NNS&q=AI+%EC%9D%98%EB%A3%8C%EA%B8%B0%EA%B8%B0
- https://www.iec.ch/dyn/www/f?p=103:14:715051018849207::::FSP_ORG_ID,FSP_LANG_ID:28235,25
- 한국 제안으로 AI/ML-MD에 대한 performance evaluation process 최초 표준 개발 시작 (2025년 제정 예상) 
- 30개국 만장일치로 IEC TC62 산하에 PT8 신설과 의장단 수임
- AI/ML 시스템에 대한 품질 모델, 신뢰성, 데이터, 테스팅 등등에 대해 여러 표준화 기구에서 경쟁적인 표준 개발이 진행 중임. (점점 영역에 대해 정리가 되는 중)   
- AI/ML-MD의 quality model에서 가장 중요한 것은 performance이며, performance에 대한 체계적 평가 절차 표준 
- performance에 대해 3 pillar 모델로 평가 
- 2월에 Testing for AI systems 표준 개발이 시작되면, 중요한 연계 체계도 만들어질 것으로 기대 
- 관심 있는 분들은 언제든 연락주세요

**Comment by priancho** [comment]
Diffusion model에 대한 fast inference 관련해서 아래 논문을 본 기억이 있어서 공유합니다.
https://arxiv.org/pdf/2110.02037.pdf

**Comment by kwonminki** [comment]
아까 말씀드린 논문은 denosing deffusion gan으로 nvidia에서 이번에 낸 논문입니다.
https://nvlabs.github.io/denoising-diffusion-gan/
https://arxiv.org/abs/2112.07804

**Comment by nick-jhlee** [comment]
- 교육 of AI
  - [[톡톡에듀] 2024년 도입되는 초등 AI교육, 이건 알아둬야](https://www.joongang.co.kr/article/25019862#home)
  - [인공지능 수학 (2015년 교육과정 개정)](https://www.cmass21.co.kr/ebook/H1/ingongjineung-suhag_ebook/ingongjineung-suhag_ebook.html)
  - [유초중고 교육에 AI 도입…고교 내년 2학기 AI 과목 신설](https://www.korea.kr/news/policyNewsView.do?newsId=148880030)

**Comment by terryum** [comment]
- ["Most Time Series Anomaly Detection Papers are Wrong" (Reddit)](https://www.reddit.com/r/MachineLearning/comments/rwwjal/d_a_paper_suggests_most_time_series_anomaly/)

어떤 분이 흥미로운 논문을 발견(?)해서 1월 5일 Reddit에 공유를 했네요. Time-series anomaly detection (TAD) 문제의 evaluation 과정에서 후처리로 쓰는 point adjustment (PA)라는 프로세스가 있는데, PA만 있으면 random prediction도 SOTA를 찍는 놀라운 변신 능력(?)을 보여준다고 합니다. 즉, 성능평가에 있어 PA를 적용한 여러 TAD 연구들의 성능이 매우 과장되어 있다는 것이죠.

원 논문은 (흥미롭게도) 한국에서 나온 논문입니다. (will appear in AAAI-22)
[Towards a Rigorous Evaluation of Time-series Anomaly Detection](https://arxiv.org/abs/2109.05257)
Siwon Kim, Kukjin Choi, Hyun-Soo Choi, Byunghan Lee, Sungroh Yoon

PA는 "ground truth 입장에서 보았을 때 하나라도 point-wise prediction에 걸렸다면 모두를 맞춘 것으로 친다"라고 하는 '하나만 걸려라' 식 후처리입니다. 아래 그래프에서처럼 딱 하나만 point prediction이 threshold를 넘어도 GT를 참고해 모든 point를 맞춘것처럼 prediction을 수정해 주는거죠.
![image](https://user-images.githubusercontent.com/12528769/149661520-3660f285-8617-4f27-96fd-f5ea4bed4ee5.png)
![image](https://user-images.githubusercontent.com/12528769/149661640-4f194acf-3461-473c-8c85-223a3fc74fac.png)

우습게도(?) PA를 적용하면 현존하는 최고의 TAD 알고리즘은 random prediction이 된다고 하네요.
![image](https://user-images.githubusercontent.com/12528769/149661757-57dcafda-0059-4e37-8938-e42435b2b422.png)

Reddit의 댓글에는 time-series prediction에서 꾸준히 재기되어왔던 성능평가에 대한 이슈를 비판하며 "오늘의 날씨 = 어제의 날씨"와 같은 냉소적인 댓글이 달리고 있네요.


[Terry의 의견]
학계에서 말하는 '벤치마크에서 높은 성능을 찍었다'라는 것과 실제 현장에서 '이 알고리즘이 정말 쓸만하다'라는 것과 차이가 날 때가 많습니다. 때로는 데이터의 가정이 현실적이지 않아서, 때로는 현실에선 논문처럼 문제가 명확하게 정의되지 않아서, 때로는 evaluation이 현실문제와는 동떨어져서 등의 이유로 현실에선 못쓰이는 경우가 많죠. 하지만 여전히 많은 연구자들은 "쓸모없는 문제"에서 SOTA를 찍으려 노력합니다. 

현실에서 가장 어려운 문제 중 하나가 '이 문제를 잘 풀었다는 것을 어떻게 evaluation 하는가?'에 대한 답을 내는 것입니다. 현실문제를 수학적으로 모델링하고, 그 성능을 하나의 숫자로 표현 한다는 건 정말 어려운 일이죠 (그것만으로도 AI에서 큰 업적을 냈다고 할만합니다.) 하지만 TAD 문제가 어렵단 이유로 이렇게 관대한 metric을 2018년에 제안하고 아직까지도 써왔다는 것에 대해선 유감입니다. 아마도 논문의 우수성을 증명하고 싶었겠지만, 어려운 문제는 과장없이 '어렵다'는 결론으로 두는게 맞지 않을까 싶습니다.


[Reference] PA를 처음 evaluation에 사용한 논문:
[Unsupervised Anomaly Detection via Variational Auto-Encoder for Seasonal KPIs in Web Applications, H. Su et al. ](https://arxiv.org/abs/1802.03903)
